modules/logging-release-notes-5-9-3.adoc

```diff
@@ -0,0 +1,18 @@
+/module included in logging-5-9-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-9-3_{context}"]
+= Logging 5.9.3
+This release includes link:https://access.redhat.com/errata/RHBA-2024:3736[OpenShift Logging Bug Fix Release 5.9.3]
+
+[id="logging-release-notes-5-9-3-bug-fixes"]
+== Bug Fixes
+
+* Before this update, there was a delay in restarting ingesters when configuring `LokiStack` as the {loki-op} set the write-ahead log `replay_memory_ceiling` to zero bytes for the `1x.demo` size. With this update, the minimum value used for the `replay_memory_ceiling` has been increased to avoid delays. (link:https://issues.redhat.com/browse/LOG-5614[LOG-5614])
```

Comment:
"when configuring `LokiStack` as the {loki-op} set the write-ahead log `replay_memory_ceiling` to zero bytes for the `1x.demo` size.."

Not really sure what this means? "as the {loki-op} set the"?

Reply:
```suggestion
* Before this update, there was a delay in restarting ingesters when configuring `LokiStack` as the {loki-op} sets the write-ahead log `replay_memory_ceiling` to zero bytes for the `1x.demo` size. With this update, the minimum value used for the `replay_memory_ceiling` has been increased to avoid delays. (link:https://issues.redhat.com/browse/LOG-5614[LOG-5614])
```

Reply:
Does that fix the issue?